hidden-posts.php

```diff
@@ -180,6 +181,48 @@ public function custom_column_style() {
 		print( '<style> .fixed .column-visibility { width: 5.5em; } </style>' );
 	}
 
+	/**
+	 * Add custom filter to the admin columns.
+	 *
+	 * @param array $views The original array with view links.
+	 *
+	 * @return array The updated array with view links.
+	 */
+	public function custom_column_filter( $views ) {
+
+		if ( ! is_admin() ) {
+			return $views;
+		}
+
+		if ( isset( $_GET['post_type'] ) && 'post' !== $_GET['post_type'] ) { // phpcs:ignore
```

Comment:
I'd like to see the `phpcs:ignore` specify which rule it's ignoring.

Reply:
Good catch, @trepmal. Until now, I've only used `// phpcs:ignore` without further explanation. Now that you pointed this out, I see the benefit of adding the ignored rule to it for transparency reasons.

In the particular case that you highlighted, I added the `// phpcs:ignore` statement because of a missing nonce. As the related function only reads data, but do not write and/or edit data, I decided to ignore the missing nonce.